### DISCUSSION TITLE
Error at line 448.

### DISCUSSION THREAD

**Discussion by shiratsukee** [Lailloken/Exile-UI#221] [General]
any idea how to fix? opened it as administrator

![error](https://user-images.githubusercontent.com/36142064/230645272-18eb2080-cec8-4d07-a7bb-d01f9fef6484.png)

**Comment by Lailloken** [comment]
Your AutoHotkey installation is outdated, I think. EDIT: get the newest 1.1.x version

**Reply by shiratsukee** [reply]
> Your AutoHotkey installation is outdated, I think. EDIT: get the newest 1.1.x version

Fixed the issue, thanks a lot!
